project-management/FujitsuMeetingDates

```diff
@@ -0,0 +1,14 @@
+## Meetings Descriptions
+Each sprint is 9 days (approx. 2 work weeks long) therefore each of these meetings occurs every 2 weeks.
+*Backlog refinement* is carried out before each sprint to reprioritise the backlog of user stories (issues) ahead of the next sprint planning meeting.
+*Sprint Planning* is carried out at the start of each sprint and is when the user stories (issues) that are planned to be completed in that sprint are chosen.
+*Sprint retrospective* is carried out at the end of each sprint to consider how the sprint went and any changes that need to be made to the process before the next sprint.
+During this meeting a demo of the achievements of the previous sprint will be shown. 
+
+More details about the Agile process can be found in the Agile - Open source workflow document also in the project management folder. 
+https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/EllenDevereux-patch-2/project-management/agile-opensource-workflow.md
+
```

Comment:
Can we add a section on who is invited to the meetings? I think the goal is to have the whole Fujitsu and Turing teams and a rotating member of the autistic community, right?